pkg/util/merge/merge_test.go

```diff
@@ -182,11 +182,11 @@ func TestMergeContainer(t *testing.T) {
 		)
 		mergedContainer := Container(defaultContainer, overrideContainer)
 		assert.Equal(t, overrideContainer.Name, mergedContainer.Name, "Name was overridden, and should be used.")
-		assert.Equal(t, []string{"a", "b", "c", "d", "f", "e"}, mergedContainer.Command, "Command was specified in both, so results should be merged.")
+		assert.Equal(t, []string{"d", "f", "e"}, mergedContainer.Command, "Command was specified in both, so results should be merged.")
```

Comment:
Think we should change the comment as well to illustrate that we'd be overriding the default command.

Reply:
Good catch, fixed.